### PR TITLE
law_of_three_second_for_vacuum_diode

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests

### CODE REVIEW

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from sympy import Eq, solve
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:1]: Use descriptive names. Law of three second does not give any hint, what it is about.

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The volt-ampere characteristic of a vacuum diode is described by the law of degree three of the second. The diode
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:11]: See comment about naming in similar law.

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## constant that is in this law depends only on the relative position, shape and size of the electrodes of the vacuum diode.
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:12]:
```suggestion
## constant in this law depends only on the relative position, shape and size of the electrodes of the vacuum diode.
```

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Law is: I = g * U^(3 / 2), where
+## I - anode current,
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:15]: Is it same as cathode current?

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## g - diode constant,
+## U - voltage between its cathode and anode.
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:17]:
```suggestion
## U - voltage between cathode and anode.
```

File: symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py (hunk 4 above)
Comment (Zulusssss) [reply symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py:15]: > Is it same as cathode current?

No, the anode current and the cathode current are not equal.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**